backend/pkg/api/connect/service/console/service.go

```diff
@@ -93,7 +94,25 @@ func (api *Service) ListMessages(ctx context.Context, req *connect.Request[v1alp
 		}
 	}
 
-	interpreterCode, _ := lmq.DecodeInterpreterCode() // Error has been checked in validation function
+	interpreterCode, err := lmq.DecodeInterpreterCode()
+	if err != nil {
+		return apierrors.NewConnectError(
+			connect.CodeInvalidArgument,
+			err,
```

Comment:
```suggestion
			fmt.Errorf("failed decoding provided interpreter code: %w", err),
```

NIT: Just to add some context where it is failing.